asynq/decorators.py

```diff
@@ -248,17 +248,17 @@ def __get__(self, owner, cls):
 
 
 class AsyncProxyDecorator(AsyncDecorator):
-    def __init__(self, fn):
+    def __init__(self, fn, asyncio_fn=None):
         # we don't need the task class but still need to pass it to the superclass
-        AsyncDecorator.__init__(self, fn, None)
+        AsyncDecorator.__init__(self, fn, asyncio_fn)
```

Comment:
```suggestion
        AsyncDecorator.__init__(self, fn, None, asyncio_fn=asyncio_fn)
```
The second argument is `cls`, see line 197 above.